src/components/Skeleton/Skeleton.module.scss

```diff
@@ -1,3 +1,17 @@
+@import "./SkeletonVariables.module";
```

Comment:
full width did not work on text skeleton. fixed it for using it in the dialog docs

Reply:
I am using global because the rest of skeleton css is not converted to css modules